src/mmw/package.json

```diff
@@ -49,6 +49,8 @@
     "turf-random": "1.0.2",
     "underscore": "1.8.3",
     "watchify": "3.1.0",
-    "zeroclipboard": "2.2.0"
+    "zeroclipboard": "2.2.0",
+    "turf-intersect": "1.4.2",
```

Comment:
Upgrading it would certainly help, but for the sake of consistency I would recommend using `npm.sh` if possible.